src/Microsoft.AspNet.OData.Shared/GetNextPageHelper.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.AspNet.OData
     /// </summary>
     internal static partial class GetNextPageHelper
     {
-        internal static Uri GetNextPageLink(Uri requestUri, IEnumerable<KeyValuePair<string, string>> queryParameters, int pageSize)
+        internal static Uri GetNextPageLink(Uri requestUri, IEnumerable<KeyValuePair<string, string>> queryParameters, int pageSize, string skipTokenValue = null)
```

Comment:
consider generating the $skiptoken value here.

Reply:
Moved the skiptoken generation code to this.